core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/resources.js

```diff
@@ -87,33 +87,38 @@ window.confirmAndRevokeAllSelected = function (button) {
 
   if (allCheckedCheckBoxes.length === 0) {
     var nothingSelected = button.getAttribute("data-nothing-selected");
-    alert(nothingSelected);
+    dialog.alert(nothingSelected);
   } else {
     var confirmMessageTemplate = button.getAttribute("data-confirm-template");
     var confirmMessage = confirmMessageTemplate.replace(
       "%num%",
       allCheckedCheckBoxes.length
     );
-    if (confirm(confirmMessage)) {
-      var url = button.getAttribute("data-url");
-      var selectedValues = [];
-
-      for (var i = 0; i < allCheckedCheckBoxes.length; i++) {
-        var checkBox = allCheckedCheckBoxes[i];
-        var userId = checkBox.getAttribute("data-user-id");
-        var uuid = checkBox.getAttribute("data-uuid");
-        selectedValues.push({ userId: userId, uuid: uuid });
-      }
-
-      // TODO simplify when Prototype.js is removed
-      fetch(url, {
-        method: "post",
-        body: Object.toJSON
-          ? Object.toJSON({ values: selectedValues })
-          : JSON.stringify({ values: selectedValues }),
-        headers: crumb.wrap({ "Content-Type": "application/json" }),
-      }).then(() => window.location.reload());
-    }
+    dialog.confirm(confirmMessage).then(
+      () => {
+        var url = button.getAttribute("data-url");
+        var selectedValues = [];
+
+        for (var i = 0; i < allCheckedCheckBoxes.length; i++) {
+          var checkBox = allCheckedCheckBoxes[i];
+          var userId = checkBox.getAttribute("data-user-id");
+          var uuid = checkBox.getAttribute("data-uuid");
+          selectedValues.push({ userId: userId, uuid: uuid });
+        }
+
+        // Workaround prototype.js breaking JSON.stringify
+        // Given this page is isolated, it's safe to not restore these after to the prototype version
```

Comment:
looks like a bad merge
```suggestion
                // TODO simplify when Prototype.js is removed
```